src/cadence/scripts/getNFTIDs.cdc

```diff
@@ -167,13 +167,13 @@ pub fun main(ownerAddress: Address): {String: [UInt64]} {
     }
 
     if let col = owner.getCapability(KOTD.CollectionPublicPath)
```

Comment:
Can we remove lines 169-176? Just realized these are duplicate lines (KOTD and Crave are already accounted for here: Line 60 and 84